nsdb-cluster/src/main/resources/nsdb.conf

```diff
@@ -71,6 +71,8 @@ nsdb {
     mode = ${?CLUSTER_MODE}
     required-contact-point-nr = 1
     required-contact-point-nr = ${?CONTACT_POINT_NR}
+    write-consistency = "all"
```

Comment:
I would rename this property into `metadata-write-consistency`, since there are already properties that model a more generic consistency (in terms of number of replicas)